### PR TITLE
Removed whitespaces and typo fix

### PR DESCRIPTION
**Pull request by mmphego**
This PR removes any whitespaces from the config sample file as well as fix a typo.

### CODE REVIEW

**Comment by MichMich** [comment]
Let's not go into the tabs vs spaces discussion. ;) Also the changes contain language errors.

File: config/config.js.sample [hunk 1]
```diff
@@ -3,97 +3,97 @@
  * By Michael Teeuw http://michaelteeuw.nl
  * MIT Licensed.
  *
- * For more information how you can configurate this file
+ * For more information how you can configuration this file
```
Comment (MichMich) [inline config/config.js.sample]: This doesn't seem correct.

File: config/config.js.sample [hunk 2]
```diff
@@ -3,97 +3,97 @@
...
  * See https://github.com/MichMich/MagicMirror#configuration
  *
  */
 
 var config = {
-	address: "localhost", // Address to listen on, can be:
-	                      // - "localhost", "127.0.0.1", "::1" to listen on loopback interface
-	                      // - another specific IPv4/6 to listen on a specific interface
-	                      // - "0.0.0.0", "::" to listen on any interface
-	                      // Default, when address config is left out or empty, is "localhost"
```
Comment (MichMich) [inline config/config.js.sample]: It can also be empty. Why remove this?

Comment (mmphego) [reply config/config.js.sample]: Figured left out and empty more or less meant the same thing. I'll revert the change. Thanks

File: config/config.js.sample (hunk 1 above)
Comment (mmphego) [reply config/config.js.sample]: I completely butchered the Queen's language. Will fix it on next commit.

**Comment by MichMich** [comment]
Thanks for the updates! I think every other file is using tabs. So we might want to use tabs in the configuration as well.

**Comment by mmphego** [comment]
Instead of messing up the commit history, I will close this PR and create a new one with only the grammar fixes :+1:

**Pull request closed without merging**